core/src/main/java/bio/terra/pearl/core/model/participant/EnrolleeRelation.java

```diff
@@ -13,17 +14,21 @@
 import lombok.Setter;
 import lombok.experimental.SuperBuilder;
 
+/** represents a relation between two enrollees.
+ * this relationship is directional from the enrollee to the
+ * targetEnrollee
+ * To read semantically,  "The enrollee with id `enrolleeId` is the `relationshipType` of/to the enrollee with id `targetEnrolleeId`
+ * e.g.  if enrolleeId is 1, and targetEnrolleeId is 2, 1 is the proxy of 2. (not vice-versa)
```

Comment:
To be consistent with the new field on `Enrollee`, how would you feel about renaming `targetEnrolleeId ` to `subjectEnrolleeId`? Or is there some more nuance here which warrants the more generic wording?

Reply:
that's correct -- the target might be a subject, or it might be someone who is not a subject.  E.g. the relationship could be a family member of a non-subject